code/game/objects/items/ego_weapons/special.dm

```diff
@@ -153,11 +155,13 @@
 				sleep(3)
 			if(3)
 				sleep(2)
-		target.apply_damage(force, damtype, null, target.run_armor_check(null, damtype), spread_damage = TRUE)
-		user.do_attack_animation(target)
-		playsound(loc, hitsound, 30, TRUE, extrarange = stealthy_audio ? SILENCED_SOUND_EXTRARANGE : -1, falloff_distance = 0)
-		playsound(loc, 'sound/abnormalities/we_can_change_anything/change_generate.ogg', get_clamped_volume(), FALSE, extrarange = stealthy_audio ? SILENCED_SOUND_EXTRARANGE : -1, falloff_distance = 0)
-		new /obj/effect/temp_visual/dir_setting/bloodsplatter(get_turf(target), pick(GLOB.alldirs))
+		var/multihit = force * justicemod * force_multiplier
+		if(target in view(reach,user))
+			target.apply_damage(multihit, damtype, null, target.run_armor_check(null, damtype), spread_damage = TRUE)
```

Comment:
Update your fork to upstream/master, then
We got a new proc to use here, instead of `target.apply_damage` you can do `target.deal_damage`
`deal_damage()` automatically does an armor check of the type of damage you are dealing, along with spreading it.